### DISCUSSION TITLE
If the same ID is inserted but no error is reported, is it overwritten or something else?

### DISCUSSION THREAD

**Discussion by yesletgo** [milvus-io/milvus#32415] [Q&A and General discussion]
In the new version 2.4.0, if the same ID is inserted but there is no error, is it overwritten, or something else？

**Comment by yhmo** [comment]
Milvus doesn't report error for duplicate IDs. If you insert multiple items with the same ID, Milvus will store all of them in storage.
When you call search and the results contain the same ID items, there will only be one item in the results. Usually, the only item is the latest inserted item.
